tools/webpack/packages.js

```diff
@@ -302,6 +328,7 @@ module.exports = function( env = { environment: 'production', watch: false, buil
 					...cssCopies,
 					...phpCopies,
 					...blockMetadataCopies,
+					...blockStylesheetCopies,
```

Comment:
This adds 260 CSS files to subfolders in [src/wp-includes/blocks](https://github.com/WordPress/wordpress-develop/tree/master/src/wp-includes/blocks). It won't scale well because they can change not only when the content of files updates but also when we modify webpack configuration. In this folder, we also have `block.json` files and PHP files copied over from WordPress packages with webpack. The only file that is custom is inside `index.php`. Ideally, we should remove all files inside `src/wp-include/blocks` from tracking in svn and leave it to the build process. I think it was proposed by @peterwilsoncc in https://core.trac.wordpress.org/ticket/49635.